curriculum/challenges/english/08-coding-interview-prep/data-structures/perform-a-union-on-two-sets.english.md

```diff
@@ -12,7 +12,7 @@ For example, if <code>setA = ['a','b','c']</code> and <code>setB = ['a','b','d',
 
 ## Instructions
 <section id='instructions'>
-
+Create a method <code>union</code>. This method takes another <code>Set</code> as an argument and returns a <em>new</em> Set. 
```

Comment:
@ezioda004, I just left a comment in #34492 saying that I don't think we should merge the challenges. I believe that combining the challenges here could make them too complicated to complete in about two minutes, and users might be frustrated by tests that seem to compete--create a function performs a union on two sets of data and another that performs an intersection on two sets. They just seem to be pulling in two different directions at once.

I'm just thinking that it would be fine to remove the text for the instructions here. The description explains the challenge clearly, so the instructions explaining the same thing seem redundant.

But like I said in #34492, I could see adding more tests, or editing the text of the existing tests or descriptions so that they feel more like substantial challenges like you mentioned.

That's all just my opinion, though. Did you have some ideas of how you would like to merge these challenges?

Reply:
Hey @scissorsneedfoodtoo,
Apologizes for the late reply, I was travelling the whole day yesterday.
 
>Why don't we just remove the text for the instructions here and update the text of the test below?

Yeah thats sounds good. We can then merge these 3 challenges into one.